paas/smoke-test-apps.yml

```diff
@@ -34,5 +34,8 @@ applications:
       NOTIFICATIONS_URL: ((notifications_url))
       DIRECT_DEBIT_GOCARDLESS_WEBHOOK_SECRET: ((direct_debit_gocardless_webhook_secret))
 
+  - <<: *smoke-test-app
```

Comment:
Kind of think this could be one app definition with 
`name: smoke-tests-((environment))` couldn't it? We don't push the entire manifest of all apps, only ever one referenced by name? https://github.com/alphagov/pay-omnibus/blob/master/ci/pipelines/deploy.yml#L1034